m1/arm64.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM adoptopenjdk/openjdk11:jre-11.0.14_9-ubuntu
```

Comment:
Could we use the eclipse-temurin image here, to be consistent with what we are doing with Java 17? (this was @heathd's suggestion)

Reply:
Ah wait, maybe it's just that they are keeping version 11 in the old org?

Reply:
There is an eclipse-temurin 11 image, I think you're right that we should aim to be consistent even if these are dev images only. Have changed to `eclipse-temurin:11.0.14_9-jre`.